src/script/interpreter.cpp

```diff
@@ -1257,15 +1234,6 @@ uint256 SignatureHash(const CScript& scriptCode, const CTransaction& txTo, unsig
         }
     }
 
-#if 1
```

Comment:
remove the same check in `signrawtransaction`

Reply:
Done